src/scripts/main.js

```diff
@@ -1,3 +1,19 @@
 'use strict';
 
-// write code here
+const tree = document.querySelector('.tree');
+const headers = document.querySelectorAll('li');
+
+for (const header of headers) {
+  const span = document.createElement('span');
+
+  header.prepend(span);
+  span.append(span.nextSibling);
+}
+
+tree.addEventListener('click', (e) => {
+  if (e.target.closest('span').nextSibling.style.display === 'none') {
```

Comment:
you can move `e.target.closest('span').nextSibling.style.display` to a variable to avoid repeating yourself.
And then just use ternary operator